third-party/picocom/impl.h

```diff
@@ -0,0 +1,20 @@
+
+#define _POSIX_PATH_MAX 256
```

Comment:
it should be placed in <limits.h> directly

Reply:
will do.

Reply:
I pushed one commit modifying limits.h and adding the POSIX/SUS minimum values required for any POSIX system. this include _POSIX_PATH_MAX, and the commit itself is inline with eventually making the system POSIX compliant.